polkadot/runtime/common/src/try_runtime.rs

```diff
@@ -36,7 +36,7 @@ where
 
 	let all_stakers = Ledger::<T>::iter().map(|(ctrl, l)| (ctrl, l.stash)).collect::<BTreeSet<_>>();
 	let mut all_exposed = BTreeSet::new();
-	ErasStakersPaged::<T>::iter().for_each(|((_era, val, _page), expo)| {
+	ErasStakers::<T>::iter().for_each(|(_, val, expo)| {
```

Comment:
This shouldn't be reverted. 
```suggestion
	ErasStakersPaged::<T>::iter().for_each(|((_era, val, _page), expo)| {
```